LabApiUtilities/build.gradle

```diff
@@ -36,7 +36,7 @@ sourceSets {
         java.srcDirs = ['src/main']
     }
     test {
-        java.srcDirs = ['src/test', 'build/generated/source/buildConfig/test']
+        java.srcDirs = ['src/test', "$project.buildDir/generated/source/buildConfig/test"]
```

Comment:
Maybe add {} to make the reference clearer.

Reply:
didn't seem to work, and I also didn't see us using it anywhere in the project..keeping unchanged